tests/sentry/integrations/gitlab/test_integration.py

```diff
@@ -26,13 +26,20 @@ class GitlabIntegrationTest(IntegrationTestCase):
     }
     default_group_id = 4
 
+    def setUp(self):
+        super(GitlabIntegrationTest, self).setUp()
+        self.init_path_without_guide = '%s%s' % (self.init_path, '?completed_installation_guide')
+
     def assert_setup_flow(self, user_id='user_id_1', group_id=None):
         if group_id is None:
             group_id = self.default_group_id
 
         resp = self.client.get(self.init_path)
         assert resp.status_code == 200
-        resp = self.client.post(self.init_path, data=self.config)
+        self.assertContains(resp, 'you will need to create a Sentry app in your GitLab instance')
```

Comment:
@markstory this is pretty brittle. Any thoughts?

Reply:
I personally do this. If the copy changes significantly it could be an indication that the page is 'broken'. Another approach you can use is to look for specific CSS classes.